### ISSUE TITLE
Small instantiation of oulipo.social's WWW contains invalid glyph

### ISSUE THREAD

**Issue by blinry** [rjl20/mastodon#9]
In small instantiations of oulipo.social's WWW (for smart communicators which fit in your hand), old toots (> 12 months) display an invalid glyph.

I'd say you could swap it for "12 months ago", or "2017".

**Comment by rjl20** [comment]
So, for the record, it took me over a week to find this one because I was looking in the mobile view of the main interface, like at https://oulipo.social/web/accounts/903, and the "last year" text only shows up if you're looking at an individual user's page, such as https://oulipo.social/@liam?max_id=75296.

Thanks for the report! But in the future, please use the letter "e" if it will clarify where the offending string can be found.

And I'm not sure I can fix this one, at least not until I understand the internals a whole lot better. It's not a localization thing, which would be easy, or even a template thing, which would be slightly less easy but I'm getting the hang of it. The dates are being rewritten entirely in javascript somehow -- how they show up when you inspect an element is not how they show up when you view the source. I have no idea where that's controlled.

But I'll look into it.

**Comment by rjl20** [comment]
Ah, I see where it's coming from. It's part of https://github.com/yahoo/intl-relativeformat

I'm not sure I can modify that locally without my changes getting overwritten if that package gets an update, since it's an external library, not part of the mastodon code. I'll do some reading on how this all ties together, though.
